common/src/main/java/com/microsoft/identity/common/internal/commands/SignInResendCodeCommand.kt

```diff
@@ -0,0 +1,57 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.commands
+
+import com.microsoft.identity.common.internal.controllers.NativeAuthMsalController
+import com.microsoft.identity.common.java.commands.parameters.nativeauth.SignInResendCodeCommandParameters
+import com.microsoft.identity.common.java.controllers.results.SignInResendCodeCommandResult
+import com.microsoft.identity.common.java.logging.LogSession
+import com.microsoft.identity.common.java.logging.Logger
+
+class SignInResendCodeCommand(
+    private val parameters: SignInResendCodeCommandParameters,
+    private val controller: NativeAuthMsalController,
+    publicApiId: String
+) : BaseNativeAuthCommand<SignInResendCodeCommandResult>(
+    parameters,
+    controller,
+    publicApiId
+) {
+
+    companion object {
+        private val TAG = SignInResendCodeCommand::class.java.simpleName
+    }
+
+    override fun execute(): SignInResendCodeCommandResult {
+        LogSession.logMethodCall(TAG, "${TAG}.execute")
+        val result = controller.signInResendCode(
+            parameters = parameters
+        )
+
+        Logger.info(
+            TAG,
+            "Returning result: $result"
```

Comment:
So we are logging the whole result object?

Reply:
Same question for all the places result object is being logged

Reply:
@SaurabhMSFT Are you certain there is no PII in there? CommandResult holds generic `T` result and for acquire token operations that includes the `ILocalAuthenticationResult` which inside contains the account record and the tokens.

Reply:
The class com.microsoft.identity.common.java.controllers.results.ICommandResult does not have any PII. The class you mentioned is not used here. There might be confusion as the project has two classes with same name.

Reply:
Please rename the new class you've added. It is certainly causing a lot of confusion

Reply:
Done